tf/environments/dev/main.tf

```diff
@@ -230,6 +234,13 @@ resource "aws_codestarconnections_connection" "ooniapi" {
   depends_on = [module.adm_iam_roles]
 }
 
+resource "aws_codestarconnections_connection" "oonith" {
```

Comment:
I wonder if we actually need multiple codestar connections or if we should just consolidate all them into a single one.

The annoying thing of having multiple ones is that AFAIK for each codestar connection there is a manual step in the AWS console (see the comment above the `aws_codestarconnections_connection.ooniapi` resource), so perhaps we can just use a single one.

Reply:
yes, I think it makes sense to having a single codestar connection. We can probably replace this with a single 
```
resource "aws_codestarconnections_connection" "ooni" {
```
and use it for all the services (including `ooniapi` which has a separate one right now)

Reply:
Yeah. I would actually suggest calling `oonidevops` so that it's clear that it's related to this code repo.

When you do the rename be sure to add the `moved` blocks before you apply so that it doesn't trigger a destroy and recreate, but rather becomes just a `noop`: https://developer.hashicorp.com/terraform/language/modules/develop/refactoring#moved-block-syntax